svelte.config.js

```diff
@@ -1,15 +1,27 @@
 import preprocess from 'svelte-preprocess';
 import autoprefixer from 'autoprefixer';
+import { readFileSync } from 'fs';
+
+const { version } = JSON.parse(readFileSync(new URL('package.json', import.meta.url), 'utf8'));
```

Comment:
This is bloated, you should be able to do
```js
import pkg from './package.json';

const version = pkg.version;
```

> NOTE: named imports in esm do not work on json files hence not doing `import { version } ...`

Reply:
See this: https://kit.svelte.dev/faq#read-package-json -- `You cannot directly require JSON files, since SvelteKit expects svelte.config.js to be an ES module.`

Reply:
Weird, no worries then